css/sheet.css

```diff
@@ -4,20 +4,77 @@ body{
 }
 header{
     margin:0;
-    padding:0;
+   padding:0;
    text-align: center;
    width:100%;
 }
-img{
+section{
+    background: #2E2E2E;
+    width:100%;
+    padding:0;
     margin:0;
-    padding-top:5px
+    /* height:800px; */
 }
 .bg-black{
       background: rgb(0, 0, 0);
 }
 .bg-white{
     background:rgb(246, 241, 241);
 }
+.container{
+    margin:auto;
+    display:flex;
+    padding:0;
+    width:1200px;
+    align-items:center;
+    padding-top:30px;
```

Comment:
Let `.container` keeps only the properties I mentioned in Trello. Typically: width, margin and display